uxarray/conventions/ugrid.py

```diff
@@ -0,0 +1,323 @@
+    "long name": "Latitude of the center of each edge",
+    "units": "degrees_north",
+}
+
+FACE_COORDINATES = ["face_lon", "face_lat"]
+
+FACE_LON_ATTRS = {
+    "standard_name": "longitude",
+    "long name": "Longitude of the center of each face",
+    "units": "degrees_east",
+}
+
+FACE_LAT_ATTRS = {
+    "standard_name": "latitude",
+    "long name": "Latitude of the center of each face",
+    "units": "degrees_north",
+}
+
+# coordinates (cartesian)
+NODE_X_ATTRS = {
+    "standard_name": "x",
+    "long name": "x location of the corner nodes of each face ",
+    "units": "meters",
+}
+
+NODE_Y_ATTRS = {
+    "standard_name": "y",
+    "long name": "y location of the corner nodes of each face",
+    "units": "meters",
+}
+
+NODE_Z_ATTRS = {
+    "standard_name": "z",
+    "long name": "z location of the corner nodes of each face",
+    "units": "meters",
+}
+
+EDGE_X_ATTRS = {
+    "standard_name": "x",
+    "long name": "x location of the center of each edge",
+    "units": "meters",
+}
+
+EDGE_Y_ATTRS = {
+    "standard_name": "y",
+    "long name": "y location of the center of each edge",
+    "units": "meters",
+}
+
+EDGE_Z_ATTRS = {
+    "standard_name": "z",
+    "long name": "z location of the center of each edge",
+    "units": "meters",
+}
+
+FACE_X_ATTRS = {
+    "standard_name": "x",
+    "long name": "x location of the center of each face",
+    "units": "meters",
+}
+
+FACE_Y_ATTRS = {
+    "standard_name": "y",
+    "long name": "x location of the center of each face",
+    "units": "meters",
+}
+
+FACE_Z_ATTRS = {
+    "standard_name": "z",
+    "long name": "z location of the center of each face",
+    "units": "meters",
+}
+
+# connectivity (face_)
+FACE_NODE_CONNECTIVITY_ATTRS = {
+    "cf_role": "face_node_connectivity",
+    "long name": "Maps every face to its corner nodes.",
+    "start_index": 0,
+    "_FillValue": INT_FILL_VALUE,
+    "dtype": INT_DTYPE,
+}
+FACE_NODE_CONNECTIVITY_DIMS = ["n_face", "n_max_face_nodes"]
+
+FACE_EDGE_CONNECTIVITY_ATTRS = {
+    "cf_role": "face_edge_connectivity",
+    "long name": "Maps every face to its edges.",
+    "start_index": 0,
+    "_FillValue": INT_FILL_VALUE,
+    "dtype": INT_DTYPE,
+}
+FACE_EDGE_CONNECTIVITY_DIMS = [
+    "n_face",
+    "n_max_face_edges",
+]  # n_max_face_edges equiv to n_max_face_nodes
+
+FACE_FACE_CONNECTIVITY_ATTRS = {
+    "cf_role": "face_face_connectivity",
+    "long name": "Faces that neighbor each face.",
+    "start_index": 0,
+    "_FillValue": INT_FILL_VALUE,
+    "dtype": INT_DTYPE,
+}
+FACE_FACE_CONNECTIVITY_DIMS = [
+    "n_face",
+    "n_max_face_faces",
+]  # n_max_face_faces equiv to n_max_face_nodes
+
+# connectivity (edge_)
+EDGE_NODE_CONNECTIVITY_ATTRS = {
+    "cf_role": "edge_node_connectivity",
+    "long name": "Maps every edge to the two nodes that it connects.",
+    "start_index": 0,
+    "dtype": INT_DTYPE,
+}
+EDGE_NODE_CONNECTIVITY_DIMS = ["n_edge", "two"]
+
+EDGE_EDGE_CONNECTIVITY_ATTRS = {
```

Comment:
How to define "neighbor" here? Do you mean that if two edges share a vertex?

Reply:
The case where two edges share a vertex is handled by `node_edge_connectivity`

This is the definiion from the MPAS conventions

![image](https://github.com/UXARRAY/uxarray/assets/67855069/a98f954e-877d-479d-aa3b-112d041aa2e1)

Section 5.2 [here ](https://mpas-dev.github.io/files/documents/MPAS-MeshSpec.pdf)explains goes more into detail. 

I'll look into a better way we should define it, or if there is any other applications other than tangential velocities.